release_notes.md

```diff
@@ -2,5 +2,6 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR/#issue)
 -->
+- Added support for binding of `Guid` and `Guid?` type parameters.
```

Comment:
Can you add `(#582)` to the end of yours? and `(#583)` for the other one?

Reply:
Done.